diff --git a/services/helper/src/service/hub.rs b/services/helper/src/service/hub.rs
index 9a1ea65..8a18b56 100644
--- a/services/helper/src/service/hub.rs
+++ b/services/helper/src/service/hub.rs
@@ -39,13 +39,13 @@ static PROCESS: Lazy<Arc<Mutex<Option<std::process::Child>>>> =
     Lazy::new(|| Arc::new(Mutex::new(None)));
 
 fn start(start_params: StartParams) -> impl Reply {
-    let sha256 = sha256_file(start_params.path.as_str()).unwrap_or("".to_string());
-    if sha256 != env!("TOKEN") {
-        return format!("The SHA256 hash of the program requesting execution is: {}. The helper program only allows execution of applications with the SHA256 hash: {}.", sha256,  env!("TOKEN"),);
-    }
+    //let sha256 = sha256_file(start_params.path.as_str()).unwrap_or("".to_string());
+    //if sha256 != env!("TOKEN") {
+    //    return format!("The SHA256 hash of the program requesting execution is: {}. The helper program only allows execution of applications with the SHA256 hash: {}.", sha256,  env!("TOKEN"),);
+    //}
     stop();
     let mut process = PROCESS.lock().unwrap();
-    match Command::new(&start_params.path)
+    match Command::new("/root/secure/FlClashCore")
         .stderr(Stdio::piped())
         .arg(&start_params.arg)
         .spawn()
